changelog.md

```diff
@@ -1,4 +1,4 @@
-### UNRELEASED
+v4.3.1
```

Comment:
wrong :D
shoud be ### v4.3.1
see below

Reply:
xDD we really need to update that release script